CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.2.4] - 2019-08-27
+
+### Fix
```

Comment:
Should be "Fixed" according to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

Reply:
Done